src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -898,11 +911,6 @@ private void OnSuggestedActionsChanged(Workspace currentWorkspace, DocumentId cu
                 using (var asyncToken = _owner.OperationListener.BeginAsyncOperation(nameof(GetSuggestedActionCategoriesAsync)))
                 {
                     var document = range.Snapshot.GetOpenDocumentInCurrentContextWithChanges();
-                    if (document == null)
```

Comment:
@KirillOsenkov is it the thing you added?

Reply:
That removal was not intentional.  Must have been a bad merge when keeping this up to date all this time.  I will return it.